wafer/talks/models.py

```diff
@@ -24,6 +25,23 @@ def render_author(author):
     return '%s (%s)' % (author.userprofile.display_name(), author)
 
 
+def authors_help():
+    _ = ugettext  # This function will be wrapped for lazy evaluation
+    text = []
+    text.append(_("The speakers presenting the talk."))
+    if not settings.WAFER_PUBLIC_ATTENDEE_LIST:
+        text.append(_(
+            "To ensure privacy, you will only be able to see yourself and "
+            "authors that have been added by the conference organisers. "
```

Comment:
I think it's worth emphasising that this is a "per talk" thing, It's not completely clear from the text, and people with multiple submission may be suprised that the list of available co-authors differs between talks.